webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path")
+
+const { NODE_ENV = "development" } = process.env
+
+module.exports = {
+    mode: NODE_ENV,
+    devtool: "sourcemap",
+    entry: "./src/index.ts",
+    stats: "errors-only",
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
+                use: "ts-loader",
```

Comment:
Loaders are plugins that compile different file types. `ts-loader` is what handles typescript transpilation, which is then passed back to webpack which stitches everything together into a browser-friendly single file.